File: java-app/src/hr/algebra/model/Route.java
package hr.algebra.model;

/**
 *
 * @author efurkev
 */
public class Route {

    public int idRoute;
    public int timeHours;
    public int startCityId;
    public int stopCityId;
    public int kilometers;
    public int avgSpeed;
    public int fuelUsed;

    public Route(int timeHours, int startCityId, int stopCityId, int kilometers, int avgSpeed, int fuelUsed) {
        this.timeHours = timeHours;
        this.startCityId = startCityId;
        this.stopCityId = stopCityId;
        this.kilometers = kilometers;
        this.avgSpeed = avgSpeed;
        this.fuelUsed = fuelUsed;
    }

    public Route(int idRoute, int timeHours, int startCityId, int stopCityId, int kilometers, int avgSpeed, int fuelUsed) {
        this(timeHours, startCityId, stopCityId, kilometers, avgSpeed, fuelUsed);
        this.idRoute = idRoute;
    }

    public int getIdRoute() {
        return idRoute;
    }

    public void setIdRoute(int idRoute) {
        this.idRoute = idRoute;
    }

    public int getTimeHours() {
        return timeHours;
    }

    public void setTimeHours(int timeHours) {
        this.timeHours = timeHours;
    }

    public int getStartCity() {
        return startCityId;
    }

    public void setStartCity(int startCityId) {
        this.startCityId = startCityId;
    }

    public int getStopCity() {
        return stopCityId;
    }

    public void setStopCity(int stopCityId) {
        this.stopCityId = stopCityId;
    }

    public int getKilometers() {
        return kilometers;
    }

    public void setKilometers(int kilometers) {
        this.kilometers = kilometers;
    }

    public int getAvgSpeed() {
        return avgSpeed;
    }

    public void setAvgSpeed(int avgSpeed) {
        this.avgSpeed = avgSpeed;
    }

    public int getFuelUsed() {
        return fuelUsed;
    }

    public void setFuelUsed(int fuelUsed) {
        this.fuelUsed = fuelUsed;
    }

    @Override
    public String toString() {
        return "Route{" + "idRoute=" + idRoute + ", timeHours=" + timeHours + ", startCityId=" + startCityId + ", stopCityId=" + stopCityId + ", kilometers=" + kilometers + ", avgSpeed=" + avgSpeed + ", fuelUsed=" + fuelUsed + '}';
    }
}

File: dot-net-app/Ishod1/Model/Route.cs
﻿namespace Ishod1.Model
{
    class Route
    {
        public int IDRoute { get; set; }
        public int TimeHours { get; set; }
        public City CityA { get; set; }
        public City CityB { get; set; }
        public int Kilometers { get; set; }
        public int AvgSpeed { get; set; }
        public int FuelUsed { get; set; }

        public Route() { }

        public Route(int time, City cityA, City cityB, int kilometers, int avgSpeed, int fuelUsed)
        {
            TimeHours = time;
            CityA = cityA;
            CityB = cityB;
            Kilometers = kilometers;
            AvgSpeed = avgSpeed;
            FuelUsed = fuelUsed;
        }

        public Route(int idRoute, int time, City cityA, City cityB, int kilometers, int avgSpeed, int fuelUsed)
            : this(time, cityA, cityB, kilometers, avgSpeed, fuelUsed) => IDRoute = idRoute;

        public override string ToString() => $"{CityA} -> {CityB}";
    }
}
